src/events/pollVotationResults.js

```diff
@@ -8,51 +8,72 @@ module.exports = {
   async execute(message) {
     const { embeds, author, client, channelId } = message;
 
-    if (
-      !author.bot ||
-      !Array.isArray(embeds[0]?.fields) ||
-      embeds[0]?.data?.type !== 'poll_result'
-    ) {
+    if (!author.bot || !embeds || embeds[0]?.data?.type !== 'poll_result') {
```

Comment:
this conditional was removed, but that was for avoid bot crash with fake poll results, please check if this change is intentional or is an error
```
!Array.isArray(embeds[0]?.fields) ||
````

Reply:
I think you miss to add again: `!Array.isArray(embeds[0]?.fields) ||` in this conditional